rpki-validator/src/main/resources/packaging/generic/conf/application.properties

```diff
@@ -0,0 +1,94 @@
+#     this list of conditions and the following disclaimer.
+#   - Redistributions in binary form must reproduce the above copyright notice,
+#     this list of conditions and the following disclaimer in the documentation
+#     and/or other materials provided with the distribution.
+#   - Neither the name of the RIPE NCC nor the names of its contributors may be
+#     used to endorse or promote products derived from this software without
+#     specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+
+#
+# This is the port for the API, health checks, and future UI
+#
+# Note that we do not recommend exposing this externally. If you must,
+# then we recommend that an HTTPS proxy is used, and access is restricted
+#
+# We plan to include example config for such a proxy setup in future
+server.port=8080
+
+################
+# Use the following settings to change JVM parameters
+#
+# Change the initial and maximum memory for the JVM
+#
+# Notes:
+# - 512 megabytes of memory is needed for the current size of the combined RPKI repositories
+# - You may want to raise this value if you see 'out of memory' errors in the log
+# - A higher maximum will allow the JVM to use more system memory and spend less time on
+#   garbage collection (slight speed improvements possible)
+jvm.memory.initial=512m       # -Xms jvm option -> initial memory claimed by the jvm
+jvm.memory.maximum=512m       # -Xmx jvm option -> maximum memory for the jvm
+
+#
+# The following directive is used to set where the rpki-validator keeps its
+# database.
+#
+# Note that the database cannot be shared by multiple instances
+spring.datasource.url=jdbc:h2:file:./db/rpki-validator.h2
+
+#
+# The following directive is used to set where preconfigured TALs that ship with
+# the RPM can be found. This is only used when the rpki-validator-3 is first started.
+# If you wish to add (or remove) a TAL later, you will need to use the API for now.
+# A UI option may be added for this in future.
+rpki.validator.preconfigured.trust.anchors.directory=./preconfigured-tals
+
+#
+# The following directive is used to set where files retrieved from rsync based RPKI
+# repositories are kept.
+rpki.validator.rsync.local.storage.directory=./rsync
```

Comment:
`./data/rsync` ?

Reply:
I put all the directories directly under the unpacked project. Otherwise we would have ./data/data as well which I found odd.